docs/source/user_guide/operators.rst

```diff
@@ -155,8 +165,8 @@ can also be connected to work on a temporal subset:
     
 Evaluating Operators
 ~~~~~~~~~~~~~~~~~~~~
-With all the required inputs assigned, the fields_container can now be
-output from the operator with:
+With all the required inputs assigned, the ``fields_container`` can be
```

Comment:
Please provide a link to fields_container.  @cbellot000 can help here.

Reply:
@akaszynski  I think I provided a valid link.